### DISCUSSION TITLE
How Can I use it in Persian/Farsi Language?

### DISCUSSION THREAD

**Discussion by KingShah1998** [SWHL/RapidVideOCR#50] [Q&A] [answered]
How can I use it for Persian/Farsi Language? what should I do? I didn't find any program that can Extract Persian/Farsi Language ! 
Can you help me?

**Comment by SWHL** [comment]
In the PaddleOCR project, they say it supports the Persian. 
This is the related link: [docs](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_en/multi_languages_en.md#5-support-languages-and-abbreviations)

**Reply by KingShah1998** [reply]
Tnx.
How Can I use it in your program for Extracting subtitle?

**Reply by SWHL** [reply]
If you want to use it in my program, this needs a lot steps. 
When I have time, I will write a blog to instruct how to intergrate it with my program.
Please wati a moment.

**Reply by KingShah1998** [reply]
Thanks a lot. 
Then we can use it for all languages not just Persian or English language ... 
Can I contact you in telegram ? I wanna show you something.

**Reply by SWHL** [reply]
I don't have telegram account, but you can send me an email.
Email: liekkaskono@163.com

**Reply by SWHL** [reply, marked as answer]
I found I have alread written a blog about how to change the another language.
You can refer to this [article](https://rapidai.github.io/RapidOCRDocs/docs/blog/06_custom_different_model/) directly.
However, this blog is in Chinese.
